_data/people/gerritvermeulen.yml

```diff
@@ -0,0 +1,7 @@
+* Username: WebsiteGenius
+
+* Name: Gerrit Vermeulen
+
+* Orignization: Website Genius
+
+* why: I like python. But for some reason I was under the impression it was a snake. But this is cool too.
```

Comment:
So the syntax requirements here are pretty strict - you'll need to update the keys to look like:

```
username: WebsiteGenius
name: Gerrit Vermeulen
```

And so on. Make sure to remove the `*` as well.